ui/src/app/archived-workflows/components/archived-workflow-filters/archived-workflow-filters.tsx

```diff
@@ -22,7 +23,22 @@ interface ArchivedWorkflowFilterProps {
     onChange: (namespace: string, name: string, selectedPhases: string[], labels: string[], minStartedAt: Date, maxStartedAt: Date) => void;
 }
 
-export class ArchivedWorkflowFilters extends React.Component<ArchivedWorkflowFilterProps, {}> {
+interface State {
+    labels: string[];
+}
+
+export class ArchivedWorkflowFilters extends React.Component<ArchivedWorkflowFilterProps, State> {
```

Comment:
Do you think you might be able to refactor this into a functional component? They're much easier to maintain and are less buggy that class components.

Reply:
Maybe I can do this on a separate PR, will refactor to functional component by groups instead of file by file